android/src/main/java/com/imagepicker/ImagePickerModule.java

```diff
@@ -277,6 +275,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
     }
 
     response.putString("uri", uri.toString());
+    response.putString("urlPath", realPath);
```

Comment:
In my case I still needed the path to upload the image to S3. Is there a way to get a handle of the original image in Android with the uri alone?